UI/Panels/Settings/MobiFlightPanel.cs

```diff
@@ -420,6 +506,32 @@ private TreeNode mfModulesTreeView_initNode(MobiFlightModuleInfo moduleInfo, Tre
             return moduleNode;
         }
 
+        private void SetCorrectBoardIcon(TreeNode moduleNode, MobiFlightModule module)
+        {
+            if (module.Board.BoardImage != null)
+            {
+                // i had to do this with the imageKey
+                // the nodes don't support an image directly
+                var imageKey = generateImageKeyAndAddImageToImageList(module.Board);
+                moduleNode.SelectedImageKey = moduleNode.ImageKey = imageKey;
+            }
+        }
+
+        /// <summary>
+        /// This generates the imageKey and also adds the image to the imageList
+        /// </summary>
+        /// <param name="board">The Board information</param>
+        /// <returns>the valid imageKey</returns>
+        private string generateImageKeyAndAddImageToImageList(Board board)
```

Comment:
This probably has a style warning from Visual Studio, it should start with a capital letter.